docs/guide/start/managing.md

```diff
@@ -48,8 +48,12 @@ Now click on the "Sensors" tab:
 these data feeds drive the real-time picture of the application.
 As you navigate in the tree at the left, you can see more targeted statistics coming in in real-time.
 
-Explore the sensors and the tree to find a URL where the webapp we just deployed is running
-and open that in a new tab. Quickly return to the "Sensors" tab and observe the requests-per-second sensor increase.  
+Explore the sensors and the tree to find the URL where the _NginxController_ for the webapp we just deployed is running. This can be found in '**My Web Cluster** -> **My Web** -> **NginxController** -> **_main.uri_**'.
+
+_**Note**: for Softlayer/Azure/Google Compute Engine it may show the internal IP address in the URI. You will need to retrieve the public IP from the cloud provider if trying to connect to it._
```

Comment:
Currently being fixed in https://github.com/apache/incubator-brooklyn/pull/959, so it will be confusing to include this in the docs. Btw the public IP is available in `host.address` sensor.

Apart from this great changes.